kind: ConfigMap
apiVersion: v1
metadata:
  name: ibm-aiops-demo-ui-config-logs-sock
  namespace: ibm-aiops-demo-ui
data:
  DEMO_LOGS_SOCK: |-
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] dSwitch 5 - vm-network - port-1 is up.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] All is good in catalogue-db. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] All is good in catalogue-network. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] ERROR: dSwitch 5 - vm-network - port-1 is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] Port is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] ERROR: No connection.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] ERROR: No throughput","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] ERROR: network is offline. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] Port unreachable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] ERROR: No connection. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] ERROR: catalogue-db is offline. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] SQL: Database connection error. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] ERROR: No connection. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549823] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] dSwitch 5 - vm-network - port-1 is up.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] All is good in catalogue-db. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] All is good in catalogue-network. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] ERROR: dSwitch 5 - vm-network - port-1 is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] Port is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] ERROR: No connection.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] ERROR: No throughput","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] ERROR: network is offline. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] Port unreachable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] ERROR: No connection. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] ERROR: catalogue-db is offline. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] SQL: Database connection error. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] ERROR: No connection. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549883] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] dSwitch 5 - vm-network - port-1 is up.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] All is good in catalogue-db. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] All is good in catalogue-network. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] ERROR: dSwitch 5 - vm-network - port-1 is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] Port is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] ERROR: No connection.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] ERROR: No throughput","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] ERROR: network is offline. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] Port unreachable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] ERROR: No connection. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] ERROR: catalogue-db is offline. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] SQL: Database connection error. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] ERROR: No connection. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693549943] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] dSwitch 5 - vm-network - port-1 is up.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] All is good in catalogue-db. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] All is good in catalogue-network. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] ERROR: dSwitch 5 - vm-network - port-1 is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] Port is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] ERROR: No connection.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] ERROR: No throughput","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] ERROR: network is offline. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] Port unreachable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] ERROR: No connection. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] ERROR: catalogue-db is offline. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] SQL: Database connection error. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] ERROR: No connection. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550003] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] dSwitch 5 - vm-network - port-1 is up.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] All is good in catalogue-db. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] All is good in catalogue-network. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] ERROR: dSwitch 5 - vm-network - port-1 is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] Port is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] ERROR: No connection.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] ERROR: No throughput","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] ERROR: network is offline. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] Port unreachable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] ERROR: No connection. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] ERROR: catalogue-db is offline. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] SQL: Database connection error. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] ERROR: No connection. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550063] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] dSwitch 5 - vm-network - port-1 is up.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] All is good in catalogue-db. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] All is good in catalogue-network. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] ERROR: dSwitch 5 - vm-network - port-1 is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] Port is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] ERROR: No connection.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] ERROR: No throughput","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] ERROR: network is offline. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] Port unreachable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] ERROR: No connection. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] ERROR: catalogue-db is offline. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] SQL: Database connection error. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] ERROR: No connection. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550123] SQL: catalogue-db unavailable. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550183] dSwitch 5 - vm-network - port-1 is up.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550183] All is good in catalogue-db. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-network","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550183] All is good in catalogue-network. ","entities": {"pod": "catalogue-network","cluster": null,"container": "catalogue-network","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550183] ERROR: dSwitch 5 - vm-network - port-1 is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550183] Port is down.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550183] ERROR: No connection.","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "dSwitch-5-vm-network-port-1","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550183] ERROR: No throughput","entities": {"pod": "dSwitch-5-vm-network-port-1","cluster": null,"container": "dSwitch-5-vm-network-port-1","node": "dcwest1-switch025"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550183] ERROR: network is offline. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}
    {"timestamp": MY_EPOCH,"utc_timestamp": "MY_TIMESTAMP", "features": [], "meta_features": [],"instance_id": "catalogue-db","application_group_id": "1000","application_id": "1000","level": 1,"message": "[1693550183] Port unreachable. ","entities": {"pod": "catalogue-db","cluster": null,"container": "catalogue-db","node": "kube-bmgcm5td0stjujtqv8m0-ai4itsimula-wp16cpu-00002c34"},"type": "StandardLog"}


